tests/unittests/CoreGraphics.drawing/CTDrawingTests.cpp

```diff
@@ -26,7 +26,7 @@ static NSURL* __GetURLFromPathRelativeToModuleDirectory(NSString* relativePath)
 }
 #endif // WINOBJC
 
-DISABLED_DRAW_TEST_F(CTFrame, BasicDrawingTest, WhiteBackgroundTest) {
+TEXT_DRAW_TEST_F(CTFrame, BasicDrawingTest, WhiteBackgroundTest<PixelByPixelImageComparator<ComparisonMode::Mask>>) {
```

Comment:
>PixelByPixelImageComparator [](start = 64, length = 27)

I'm iffy on introducing more boilerplate for the "default" case, but if it has to be done, it has to be done